pkg/scripts/install.sh

```diff
@@ -7,8 +7,8 @@
 set -e
 CURRENT_PATH="$(pwd)"
 SCRIPT_PATH="$(dirname "$0")"
-LOG_FILE="${SCRIPT_PATH}/../../terraform-dc-install.log"
-LOG_TAGGING="${SCRIPT_PATH}/../../terraform-dc-asg-tagging.log"
+LOG_FILE="${SCRIPT_PATH}/../../terraform-dc-install_$(date '+%Y-%m-%d_%H-%M-%S').log"
```

Comment:
What is the example of the final format?

Reply:
This is gonna be the sample of output files:

```
terraform-dc-install_2021-12-03_18-34-05.log

terraform-dc-uninstall_2021-12-03_18-17-56.log
```
For each time install or uninstall script runs we will have a separated log file, with date and time of running the script.